desktop_version/src/titlerender.cpp

```diff
@@ -95,6 +95,11 @@ std::vector<std::string> changelog = {
     "  flipped)",
     "- The game no longer crashes if you give",
     "  an invalid filename to a command",
+    "- Added being able to use sprites2.png,",
+    "  sprites3.png, sprites4.png, etc.",
+    "  and using Shift+F9 to switch between",
+    "  them in the editor, not yet showing",
+    "  up in the editor",
```

Comment:
You changed the hotkey to Ctrl+F9, it's no longer Shift+F9. Update this and update FEATURES.md, too.